### PR TITLE
toy plane

### PR DESCRIPTION
**Pull request by Plyushsune**
plane
zero_but_better

## About the PR

**Media**

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: A6m toy (add more if you want)

### CODE REVIEW

**Comment by Plyushsune** [comment]
wait
what did i do

**Comment by Plyushsune** [comment]
shit?

**Comment by Plyushsune** [comment]
i pull it up to nayno once magil is back or something

**Comment by Plyushsune** [comment]
i will look into the error on friday

File: Resources/Prototypes/DeltaV/Entities/Objects/fun/zero.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ type: entity
```
Comment (FluffiestFloof) [inline Resources/Prototypes/DeltaV/Entities/Objects/fun/zero.yml]: You forgot a dash here.

Comment (Plyushsune) [reply Resources/Prototypes/DeltaV/Entities/Objects/fun/zero.yml]: thanks

**Pull request closed without merging**